packages/language-server/src/server.ts

```diff
@@ -51,13 +57,6 @@ export function startLanguageServer(connection: vscode.Connection) {
 					type: MessageType.Warning,
 				});
 			}
-
-			if (astroVersion.exist && astroVersion.major === 0 && astroVersion.minor < 24 && astroVersion.patch < 5) {
```

Comment:
This is a leftover for <Astro 0.24.5 which was released around 5-6 months ago. I think it's fine to remove it now as we head into 1.0